src/app/utilities/transmart-utilities/transmart-constraint-serialiser.ts

```diff
@@ -112,27 +150,24 @@ export class TransmartConstraintSerialiser extends AbstractConstraintVisitor<obj
     // the 'value' dimension applies to the observations with actual date values
     let dimension = constraint.isObservationDate ? 'start time' : 'value';
     let fieldName = constraint.isObservationDate ? 'startDate' : 'numberValue';
-    result = {
+    const result: TransmartTimeConstraint = {
       type: 'time',
       field: {
         dimension: dimension,
         fieldName: fieldName,
-        type: 'DATE'
+        type: TransmartType.date
       },
-      operator: operator,
+      operator: <TransmartOperator>operator,
```

Comment:
Looks like there is some miss-match between `DateOperatorState` symbols and `TransmartOperator` (before is '<=' and '<-' accordingly):
https://github.com/thehyve/glowing-bear/pull/539/files#diff-7ff27abe4af0fc943ddd37e4dcd068adR138